src/pe_source/__init__.py

```diff
@@ -0,0 +1,28 @@
+"""The pe_source library."""
+# We disable a Flake8 check for "Module imported but unused (F401)" here because
+# although this import is not directly used, it populates the value
+# package_name.__version__, which is used to get version information about this
+# Python package.
+# Standard Python Libraries
+import logging
+
+from ._version import __version__  # noqa: F401
+
+__all__ = ["cybersixgill", "shodan"]
+CENTRAL_LOGGING_FILE = "pe_reports_logging.log"
+DEBUG = False
+
+# Setup Logging
+"""Set up logging and call the run_pe_script function."""
```

Comment:
I don't see a call to `run_pe_script` - am I missing something or should this doc string be modified?
```suggestion
"""Set up logging."""
```